CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 Changelog
 =========
-1.0.1 (current)
+1.0.2 (current)
```

Comment:
please update the current one to 1.1.0. We will leave the last version number for patches.